apps/gamut-mapping/methods.js

```diff
@@ -22,6 +23,38 @@ const methods = {
 			return methods.scale.compute(mappedColor);
 		},
 	},
+	"scale-lh2": {
+		label: "Scale LH 2",
+		description: "Identical to Scale LH 2, and handles L=0/1, and noop if already in gamut.",
```

Comment:
Should this be "Identical to Scale LH" instead of "Scale LH 2"?

Reply:
Good catch! I removed this line when consolidating